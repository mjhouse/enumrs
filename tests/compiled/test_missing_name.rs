use enumrs::Tagged;

#[derive(Tagged)]
pub enum TestEnum {
    // Tags must have a name. if no expression
    // is given, the value of the tag is a bool
    // set to `true`.
    #[tag()]
    Variant = 1,
}

pub fn main() {}