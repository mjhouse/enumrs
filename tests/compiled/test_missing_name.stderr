error: No name for tag
 --> tests/compiled/test_missing_name.rs:8:5
  |
8 |     #[tag()]
  |     ^
